diff --git a/vendor/kdyby/fake-session/src/Session.php b/vendor/kdyby/fake-session/src/Session.php
index f5c5412e..5a15fbad 100644
--- a/vendor/kdyby/fake-session/src/Session.php
+++ b/vendor/kdyby/fake-session/src/Session.php
@@ -57,6 +57,13 @@ class Session extends \Nette\Http\Session
 		$this->originalSession = $originalSession;
 	}
 
+    public function autoStart(bool $forWrite): void
+    {
+        if (!$this->fakeMode) {
+            $this->originalSession->autoStart($forWrite);
+        }
+    }
+
 	public function disableNative(): void
 	{
 		if ($this->originalSession->isStarted()) {
diff --git a/vendor/kdyby/fake-session/src/SessionSection.php b/vendor/kdyby/fake-session/src/SessionSection.php
index b23c9ed9..189f2e2c 100644
--- a/vendor/kdyby/fake-session/src/SessionSection.php
+++ b/vendor/kdyby/fake-session/src/SessionSection.php
@@ -82,7 +82,7 @@ class SessionSection extends \Nette\Http\SessionSection
 	{
 	}
 
-	public function remove(): void
+	public function remove($name = null): void
 	{
 		$this->data = [];
 	}
